src/scene/gsplat/gsplat-instance.js

```diff
@@ -82,17 +99,15 @@ class GSplatInstance {
         const chunks = resource.chunks?.slice();
 
         // create sorter
-        if (!options.dither || options.dither === DITHER_NONE) {
-            this.sorter = new GSplatSorter();
-            this.sorter.init(this.orderTexture, centers, chunks);
-            this.sorter.on('updated', (count) => {
-                // limit splat render count to exclude those behind the camera
-                this.meshInstance.instancingCount = Math.ceil(count / resource.instanceSize);
-
-                // update splat count on the material
-                this.material.setParameter('numSplats', count);
-            });
-        }
+        this.sorter = new GSplatSorter();
```

Comment:
we should maybe have internal `dither` property set to false, and preserve this conditions, as it might be useful in the future?